internal/storage/blob/conf.go

```diff
@@ -35,6 +35,8 @@ type Conf struct {
 	Bucket string `yaml:"bucket" conf:"required,example=\"s3://my-bucket-name?region=us-east-2\""`
 	// Prefix specifies a subdirectory to download.
 	Prefix string `yaml:"prefix,omitempty" conf:",example=policies"`
+	// CacheDir is the local path to keep the downloaded files.
+	CacheDir string `yaml:"cacheDir" conf:",example=${HOME}/tmp/cerbos/cache"`
```

Comment:
We don't need a second directory. It would be a breaking change as well. Simply use the `WorkDir` and create two subdirectories under it. `.cache` for the files stored as etags and `store` for the symlinks.